phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Idealo For WooCommerce"
```

Comment:
Surely not so important, but do we want references to Idealo in here?

Reply:
Oh, my bad. Just copy & pasted the config with a few changes, but forgot to update this line.
Fixed.